PC02_Graffiti_Zeynalova.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Sep  8 20:28:20 2020
+
+@author: fidan
+"""
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: YOUR NAME
+May 29, 2020
+'''
+
+from turtle import * #import the library of commands that you'd like to use
+
+colormode(255)
+
+# Create a panel to draw on. 
+panel = Screen()
+w = 750 # width of panel
+h = 750 # height of panel
+panel.setup(width=w, height=h) #600 x 600 is a decent size to work on. 
+#You can experiment by making it the size of your screen or super tiny!
+
+# Create a colorful background and add Bezos image to it
+image = "Bezos.gif"
+panel.bgcolor("grey68")
+panel.bgpic(image)
+
+#=======Add your code here======
+
+#Creating turtle
+
+t = Turtle()
+t.pensize(20)
+t.shape('turtle')
+t.speed(8)
+t.pencolor(80, 80, 85)
+t.fillcolor(45, 45, 45)
+
+# Drawing a hat for the cold weather :D
+
+t.up()
+t.goto(50, 170)
+t.left(100)
+t.down()
+t.begin_fill()
+t.circle(75, 180)
+t.end_fill()
+t.left(90)
+t.goto(50, 170)
+t.left(180)
+t.pensize(30)
+t.forward(210)
+
+# Drawing the Levitating Cube
+
+t.setheading(0)
+t.up()
+t.pensize(10)
+t.pencolor(0,0,0)
+
+# Front side of the Cube is Red
+
+t.begin_fill()
```

Comment:
This block of codes seem to repeat to make a square, therefore, could add a for loop to draw the red square of the cube and add cube variable name to all the related codes that draw the cube.
```suggestion













cube.fillcolor(COLORSFILL[1])
cube.speed(8)
cube.up()
cube.goto(170, -130)
cube.down()
```